ops/terraform/env/mgmt/iam.tf

```
@@ -1,7 +1,7 @@
 resource "aws_iam_policy" "ec2_instance_tags_ro" {
   description = "Global EC2 Instances and Tags RO Policy"
   name        = "bfd-${local.env}-ec2-instance-tags-ro"
-  path        = "/"
+  path        = var.enable_cloudtamer_deployment == true ? local.cloudtamer_iam_path : "/"
```

```suggestion
  path        = local.cloudtamer_iam_path
```

Just set the path to the CloudTamer-compliant path here since it's basically a no-op in terms of actual functionality without any of the Roles having the CT Permissions Boundary attached (see above suggestion).

Please make this change for all IAM Policies/Roles that use this conditional `path` definition.